src/pages/instances/forms/UploadExternalFormatFileForm.tsx

```diff
@@ -110,7 +110,10 @@ const UploadExternalFormatFileForm: FC<Props> = ({
     const message = (
       <>
         Created instance{" "}
-        <ResourceLink type="instance" value={instanceName} to={instanceUrl} />.
+        <InstanceRichChip
+          instanceName={instanceName}
+          projectName={project?.name ?? ""}
+        />
```

Comment:
This is dropping the . and breaking tests.
```suggestion
        <InstanceRichChip
          instanceName={instanceName}
          projectName={project?.name ?? ""}
        />.
```